### PR TITLE
fix(function): remove notification for skipping deployment

### PR DESCRIPTION
**Pull request by hund030**

### CODE REVIEW

File: packages/fx-core/src/plugins/resource/function/resources/message.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { FunctionPluginPathInfo as PathInfo } from "../constants";
+import * as path from "path";
+import { FunctionPluginInfo, FunctionPluginPathInfo as PathInfo } from "../constants";
+
+export function normalizeMessage(message: string): string {
```
Comment (henzhang-ms) [inline packages/fx-core/src/plugins/resource/function/resources/message.ts:6]: We'd better not add customized format in our plugin unless we synced with solution.

File: packages/fx-core/src/plugins/resource/function/resources/message.ts [hunk 2]
```diff
@@ -14,10 +19,11 @@ export class InfoMessages {
...
   public static readonly installFunctionExtensions: string = "Installing Azure Functions binding.";
-  public static readonly noChange: string =
-    "Nothing to deploy; no changes detected since last deployment.";
-  public static readonly skipDeployment: string =
-    "Skip deployment.";
+  public static readonly noChange: string = "No changes detected since last deployment.";
+  public static readonly skipDeployment: string = `Skip deployment for no changes detected. Remove ${path.join(
+    PathInfo.solutionFolderName,
+    PathInfo.funcDeploymentFolderName
+  )} folder to force deployment.`;
```
Comment (henzhang-ms) [inline packages/fx-core/src/plugins/resource/function/resources/message.ts:26]: maybe: `To fully redeploy the function project, please remove xxx and rerun the command`.

File: packages/fx-core/src/plugins/resource/function/utils/dialog.ts [hunk 3]
```diff
@@ -1,17 +1,20 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { DialogMsg, DialogType, MsgLevel, PluginContext } from "@microsoft/teamsfx-api";
+import { PluginContext } from "@microsoft/teamsfx-api";
+import { normalizeMessage } from "../resources/message";
+
+enum MsgLevel {
```
Comment (henzhang-ms) [inline packages/fx-core/src/plugins/resource/function/utils/dialog.ts:6]: don't we have type from api?

Comment (hund030) [reply packages/fx-core/src/plugins/resource/function/utils/dialog.ts:6]: No. They didn't define such type.

File: packages/fx-core/src/plugins/resource/function/resources/message.ts (hunk 2 above)
Comment (hund030) [reply packages/fx-core/src/plugins/resource/function/resources/message.ts]: Agree. Updated.

File: packages/fx-core/src/plugins/resource/function/resources/message.ts (hunk 1 above)
Comment (hund030) [reply packages/fx-core/src/plugins/resource/function/resources/message.ts]: Good point. Removed the format for notification messages.

**Review by henzhang-ms — APPROVED** [review]

**Pull request merged**